manifester/manifester.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, manifest_category, allocation_name=None, **kwargs):
 
             self.requester = requests
             self.is_mock = False
-        self.allocation_name = allocation_name or "".join(random.sample(string.ascii_letters, 10))
+        self.allocation_name = allocation_name or f"{settings.username_prefix}-" + "".join(
+            random.sample(string.ascii_letters, 10)
```

Comment:
Just a suggestion/non-blocking comment
Nice to keep string length short, as we are going to concatinate `username_prefix` with random string of length 10, ultimately `manifest_name` string will be larger if we look at line #38

```suggestion
            random.sample(string.ascii_letters, --)
```

Reply:
This suggestion results in a syntax error when I run it locally.

To explain the reasoning for the random string, the purpose of concatenating the username prefix with the random string is to avoid collisions when one user creates more than one subscription allocation at a time on a shared RHSM account, which is a very common Manifester use case.

I'm more concerned with ensuring unique subscription allocation/manifest names than with keeping the length of this string short, but I think the length of the random string could safely be reduced to 8 characters while preserving the uniqueness. I will go ahead and make that change.